Header header
NodeState[] nodes

# name of robot
string robot_name
# group name used while launch
string launch_group
# process or launch name
string launch_config
